rosdep/python.yaml

```diff
@@ -6515,6 +6515,22 @@ python3-flask-cors:
   ubuntu:
     '*': [python3-flask-cors]
     bionic: null
+python3-flask-jwt-extended:
+  debian:
+    bullseye: [python3-python-flask-jwt-extended]
+    pip:
+      packages: [flask-jwt-extended]
+    sid: [python3-python-flask-jwt-extended]
```

Comment:
We don't add keys for sid generally due to it being a rolling/unreleased target.
```suggestion
```